contracts/access/rbac/MinterRole.sol

```diff
@@ -8,29 +8,25 @@ contract MinterRole {
 
   Roles.Role private minters;
 
-  constructor(address[] _minters) public {
-    minters.addMany(_minters);
+  constructor() public {
+    minters.add(msg.sender);
   }
 
-  function transferMinter(address _account) public {
-    minters.transfer(_account);
-  }
-
-  function renounceMinter() public {
-    minters.renounce();
+  modifier onlyMinter() {
+    require(isMinter(msg.sender));
+    _;
   }
 
   function isMinter(address _account) public view returns (bool) {
     return minters.has(_account);
   }
 
-  modifier onlyMinter() {
-    require(isMinter(msg.sender));
-    _;
+  function addMinter(address _account) onlyMinter public {
+    minters.add(_account);
```

Comment:
I forgot if we discussed this, but shouldn't there be events emitted?

Reply:
We did discuss it, we didn't include them because there's no way to reference to role itself, all we could do is `RoleAdded(account)`.